src/main/java/frc/robot/RobotContainer.java

```diff
@@ -25,31 +26,26 @@
  */
 public class RobotContainer {
   // The robot's subsystems and commands are defined here...
-  private final SwerveSubsystem swerveSubsystem = new SwerveSubsystem();
-  private final EverybotIntakeSubsystem intakeSubsystem = new EverybotIntakeSubsystem();
+  private final SwerveSubsystem swerveSubsystem = Robot.Swerve ? new SwerveSubsystem() : null;
+  private final SimulationSubsystem simulationSubsystem = Robot.isSimulation() ? new SimulationSubsystem(swerveSubsystem) : null;
 
-  private SimulationSubsystem simulationSubsystem;
-  
-  private final CameraSubsystem cameraSubsystem = new CameraSubsystem();
+  private final EverybotIntakeSubsystem intakeSubsystem = Robot.intakeSubsystem ? new EverybotIntakeSubsystem() : null;
 
-  private static BatterySubsystem batterySubsystem;
+  private final CameraSubsystem cameraSubsystem = Robot.cameraSubsystem ? new CameraSubsystem() : null;
+
+  private static BatterySubsystem batterySubsystem = Robot.batterySubsystem && !Robot.isCompetition ? new BatterySubsystem() : null;
 
   private Auton auton = new Auton(swerveSubsystem);
 
   /** The container for the robot. Contains subsystems, OI devices, and commands. */
   public RobotContainer() {
-
-    if(Robot.isSimulation()){
-      simulationSubsystem = new SimulationSubsystem(swerveSubsystem);
-    }
-    if (!Robot.isCompetition) {
-      batterySubsystem = new BatterySubsystem();
+    if (Robot.Swerve) {
```

Comment:
Should we log when a subsystem is turned on?

Just to be sure that we didn't expect it to be enabled, when it wasn't